lib/howitzer.rb

```diff
@@ -37,12 +37,13 @@ def mailgun_idle_timeout
   # @example returns url without auth
   #  app_uri.origin
 
-  def self.app_uri
+  def self.app_uri(name = nil)
```

Comment:
unit tests are missing

Reply:
If user forgot to create the settings, howitzer needs to show Warning: You used host, but forgot to define settings, please add following settings:...

Reply:
Change log should be update

Reply:
code documentation should be extended